qemu/tests/qemu_guest_agent.py

```diff
@@ -3467,6 +3495,8 @@ def get_blocks():
 
         session = self._get_session(params, None)
         self._open_session_list.append(session)
+        if params.get("os_type") == "linux":
+            self.gagent_setsebool_on(params, self.vm)
```

Comment:
And as you set selinux config, could you change them back when you finish your test?